modules/core/keeper/msg_server.go

```diff
@@ -123,7 +123,6 @@ func (k Keeper) SubmitMisbehaviour(goCtx context.Context, msg *clienttypes.MsgSu
 			clienttypes.EventTypeSubmitMisbehaviour,
 			sdk.NewAttribute(clienttypes.AttributeKeyClientID, msg.ClientId),
 			sdk.NewAttribute(clienttypes.AttributeKeyClientType, misbehaviour.ClientType()),
-			sdk.NewAttribute(clienttypes.AttributeKeyConsensusHeight, misbehaviour.GetHeight().String()),
```

Comment:
It is my understanding the `ts-relayer` does not parse misbehaviour events and I have gotten the ok from the Hermes team that handling this breaking change will be fine

I think it is good practice to handle these in-between versions breaking changes